### PR TITLE
Tests for GraphQL API

### PR DESCRIPTION
**Pull request by cichan02**
## PR-Codex overview
This PR adds GraphQL queries and mutations for countries, cities, and airports, along with JSON response files and test utility classes. It also includes a configuration class and an implementation of a JSON converter for testing purposes.

### Detailed summary
- Added GraphQL queries and mutations for countries, cities, and airports
- Added JSON response files for GraphQL queries and mutations
- Added utility classes for testing purposes
- Added configuration class for testing purposes
- Added implementation of a JSON converter for testing purposes

> The following files were skipped due to too many changes: `src/test/java/by/piskunou/solvdlaba/web/util/JsonConverter.java`, `src/main/java/by/piskunou/solvdlaba/service/fake/FakeCountryService.java`, `src/main/java/by/piskunou/solvdlaba/service/fake/FakeCityService.java`, `src/main/java/by/piskunou/solvdlaba/service/fake/FakeAirportService.java`, `src/test/java/by/piskunou/solvdlaba/web/graphql/CountryControllerTest.java`, `src/test/java/by/piskunou/solvdlaba/web/graphql/CityControllerTest.java`, `src/test/java/by/piskunou/solvdlaba/web/graphql/AirportControllerTest.java`, `src/test/java/by/piskunou/solvdlaba/web/graphql/GraphQLConfig.java`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by cichan02** [comment]
This PR adds tests for GraphQL controllers and also brings **Fakes** for services.

File: src/main/java/by/piskunou/solvdlaba/persistence/mongo/MongoAirportRepository.java [hunk 1]
```diff
@@ -2,11 +2,13 @@
 
 import by.piskunou.solvdlaba.domain.Airport;
 import java.util.regex.Pattern;
+import org.springframework.context.annotation.Profile;
 import org.springframework.data.mongodb.repository.Query;
 import org.springframework.data.mongodb.repository.ReactiveMongoRepository;
 import reactor.core.publisher.Flux;
 import reactor.core.publisher.Mono;
 
+@Profile("!test")
```
Comment (h1alexbel) [inline src/main/java/by/piskunou/solvdlaba/persistence/mongo/MongoAirportRepository.java:11]: @cichan02 create main profile instead

File: src/main/java/by/piskunou/solvdlaba/service/fake/AirportServiceFake.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+import org.jetbrains.annotations.NotNull;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+/**
+ * A fake implementation of the {@link AirportService} interface
+ * for testing purposes.
+ */
+@RequiredArgsConstructor
+public class AirportServiceFake implements AirportService {
```
Comment (h1alexbel) [inline src/main/java/by/piskunou/solvdlaba/service/fake/AirportServiceFake.java:17]: I believe that fake prefix should be first

File: src/main/java/by/piskunou/solvdlaba/service/fake/CountryServiceFake.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+import org.jetbrains.annotations.NotNull;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+/**
+ * A fake implementation of the {@link CountryService} interface
+ * for testing purposes.
+ */
+@RequiredArgsConstructor
+public class CountryServiceFake implements CountryService  {
```
Comment (h1alexbel) [inline src/main/java/by/piskunou/solvdlaba/service/fake/CountryServiceFake.java:16]: the same here

File: src/main/java/by/piskunou/solvdlaba/service/fake/CityServiceFake.java [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+import org.jetbrains.annotations.NotNull;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+/**
+ * A fake implementation of the {@link CityService} interface
+ * for testing purposes.
+ */
+@RequiredArgsConstructor
+public class CityServiceFake implements CityService {
```
Comment (h1alexbel) [inline src/main/java/by/piskunou/solvdlaba/service/fake/CityServiceFake.java:17]: the same here

**Review by h1alexbel — CHANGES_REQUESTED** [review]

File: src/main/java/by/piskunou/solvdlaba/service/fake/AirportServiceFake.java (hunk 2 above)
Comment (cichan02) [reply src/main/java/by/piskunou/solvdlaba/service/fake/AirportServiceFake.java:17]: Renamed

File: src/main/java/by/piskunou/solvdlaba/service/fake/CountryServiceFake.java (hunk 3 above)
Comment (cichan02) [reply src/main/java/by/piskunou/solvdlaba/service/fake/CountryServiceFake.java:16]: Renamed

File: src/main/java/by/piskunou/solvdlaba/service/fake/CityServiceFake.java (hunk 4 above)
Comment (cichan02) [reply src/main/java/by/piskunou/solvdlaba/service/fake/CityServiceFake.java:17]: Renamed

File: src/main/java/by/piskunou/solvdlaba/persistence/mongo/MongoAirportRepository.java (hunk 1 above)
Comment (cichan02) [reply src/main/java/by/piskunou/solvdlaba/persistence/mongo/MongoAirportRepository.java]: Checked again and removed unnecessary annotations.

**Review by h1alexbel — APPROVED** [review]

**Pull request merged**